src/lib/components/footer.svelte

```diff
@@ -23,7 +24,7 @@
           data-test-id="github-btn"
           target="_blank"
           rel="noreferrer"
-          class="transition duration-200 hover:text-primary-100 active:text-primary-100"
+          class="transition duration-200  hover:fill-red-300 hover:text-slate-400 active:text-primary-100 dark:hover:text-[#fafafa]"
```

Comment:
We are quite close I think, two very small tweaks: 

1. I think we don't need the `hover:fill-red-300`, so could you please remove that
2. Can we use `#333333` instead of `text-slate-400` for the hover state

After that it's good to go I think 🙏 👍